### DISCUSSION TITLE
Disable Chorus Plant Generation on Central End Island if Hardcore Ender Expansion or Enderlicious is Installed

### DISCUSSION THREAD

**Discussion by Rozmir-Rohi** [Roadhog360/Et-Futurum-Requiem#338] [Improvements]
Currently, a lot of the Chorus plants generated on the central End island are broken by the Ender Dragon when you fight it.
Chorus plants could instead generate solely on outer End islands from Hardcore Ender Expansion and Enderlicious if they are installed.

**Comment by Roadhog360** [comment]
Quick testing shows the vanilla dragon does not destroy the chorus trees on the central End island, although the check which prevents this is inclusive and should include ANY dragon.
If HEE's dragon is destroying the Chorus trees, this means that it is either wrongly ignoring `canEntityDestroy` in my block class or it is not an instance of the Ender Dragon class for some reason. In either case this will not be fixed because the End from 1.9 will eventually be added and I've already accounted for my own end. Making the chorus fruit not generate on other mods' central end islands is something which would take extra effort for an issue that does not happen with most dragons anyways.

I haven't tested it with HEE's dragon but I probably won't test it anyhow due to the above reason of it being incorrect handling on their end.

Either way this suggestion is invalid because it suggests to change behavior to account for something which does not happen, or a bug in another mod's dragon.
